MvvmCross/iOS/iOS/Views/Presenters/Attributes/IMvxOverridePresentationAttribute.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+namespace MvvmCross.iOS.Views.Presenters.Attributes
+{
+    public interface IMvxOverridePresentationAttribute
+    {
+        MvxBasePresentationAttribute OverridePresentationAttribute();
```

Comment:
I'm wondering if we can actually base this on a `IMvxPresentationAttribute` instead of the base one.